content/docs/reference/templates/functions.md

```diff
@@ -2221,21 +2221,20 @@ Places commas to separate groups of thousands in a number.
 
 - `number`: the number to format. Must be an int or a string. Must be a whole number.
 
-#### inFold
+#### in
 
 ```yag
-{{ $result := inFold <list> <value> }}
+{{ $result := in <sequence> <value> }}
 ```
 
-Same as [in](#in), but is case-insensitive.
+Returns whether the case-sensitive `value` is in the `sequence`. `sequence` can be a slice or string.
 
-#### in
+#### inFold
 
 ```yag
-{{ $result := in <list> <value> }}
+{{ $result := inFold <sequence> <value> }}
 ```
```

Comment:
lint: MD031: Fenced code blocks should be surrounded by blank lines

Reply:
I'm sorry, but I'm not seeing any issues with the format of the files, yet it is being flagged(?). Is there something I need to change?

Reply:
Yup, that’s right! This is part of the markdown standards many projects (including this one)
conform to. Here’s the relevant documentation about this markdown error! https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines

Reply:
Ok, sorry... I looked over it multiple times and did not notice that there wasn't a new line there. I must have deleted it by accident as I should not have been editing that line at all. I understood what the error meant, but thank you for clarifying!